doc/reverse_engineering/civilizations.md

```diff
@@ -0,0 +1,100 @@
+# Civilization attributes
+
+## Age of Empires I
+
+### Civilization specials
+
+Civilization | Bonus
```

Comment:
Thanks. I didn't notice it until I viewed the markdown files with github's webinterface.
I have corrected the markdown formatting issues and fixed some typos.